sys/fatfs_diskio/native/native_diskio.c

```diff
@@ -0,0 +1,287 @@
+/*

```

Comment:
Maybe move this file to `cpu/native` instead, since it is only relevant for native anyway